python/python/lancedb/query.py

```diff
@@ -675,13 +684,18 @@ def __init__(
         query: str,
         ordering_field_name: str = None,
         fts_columns: Union[str, List[str]] = None,
+        *,
+        use_tantivy: bool = True,
```

Comment:
Same here, we should be able to infer this parameter from index?

Reply:
Removed